css/builder.css

```diff
@@ -556,6 +556,34 @@ input[type="time"]::-webkit-calendar-picker-indicator {
   opacity: 0;
 }
 
+/* SLIDER FIELD */
+
+.form-group.fl-range-slider .range-slider .range-slider-handle,
```

Comment:
Why are these necessary? It looks like it's just overwriting the same styles that are already defined in `public/assets/fliplet-ui-rangeslider/1.0/ui.rangeslider.css` of https://github.com/Fliplet/fliplet-api/pull/5566/files

We should only be including anything that's uniquely different from the JS API.